buildmodel/commands.go

```diff
@@ -397,6 +417,14 @@ type UpdateFlags struct {
 	skipSubmoduleUpdate *bool
 }
 
+type GHAuthFlags struct {
```

Comment:
Does this really belong in `buildmodel/commands.go`? I'd expect it to live with other GitHub utilities for nice reuse.